src/components/Dashboard/DashboardGrid.story.jsx

```diff
@@ -14,7 +14,7 @@ const Cards = [
     title="Facility Metrics"
     key="facility"
     id="facility"
-    size={CARD_SIZES.SMALL}
+    size={CARD_SIZES.MEDIUMTHIN}
```

Comment:
Can we add a new DashboardGrid story with all the supported card sizes so that we can easily compare any potential changes in the future?